File: fix_packing.py
with open('FULL_PackingPage.jsx', 'r') as f:
    lines = f.readlines()
fixed = [l for l in lines if 'The above content shows the entire' not in l and not l.strip() == '']
with open('/home/user/Downloads/ssh-app/ssh-app/src/features/seed/payments/packing/PackingPage.jsx', 'w') as f:
    f.writelines(fixed)

File: fix_packing.js
const fs = require('fs');
const content = fs.readFileSync('FULL_PackingPage.jsx', 'utf-8');
const lines = content.split('\n');
const fixed = lines.filter(l => !l.startsWith('The above content shows the entire')).join('\n');
fs.writeFileSync('/home/user/Downloads/ssh-app/ssh-app/src/features/seed/payments/packing/PackingPage.jsx', fixed);
